networking/networking.md

```diff
@@ -15,7 +15,18 @@ A LinkedIn profile
 
 
 ### Motivation
-At some point in our careers, every one of us will need to ask others for feedback, connections, or support. The more you can build and manage your network, the better able you'll be to achieve your goals. With the right strategies, networking can be enjoyable and advantageous.
+At some point in our careers, every one of us will need to ask others for feedback, connections, or support. The more you can build and manage your network, the better able you'll be to achieve your goals. With the right strategies, networking can be enjoyable and advantageous. The wider the connections more is the chance to explore and express your skills. It is also important when it comes to searching for jobs. It is one of the important strategies for the advancement of one's career.
+
+Benefits of networking: 
+- Generation of referrals/Increased business
+- Opportunities
+  - With a motivated group of business owners comes an abundance of opportunities! There are always lots of opportunities that come from networking and in fact this is where the benefits of business networking are huge!
```

Comment:
Business owners?

Reply:
yes. Is anything wrong in it?

Reply:
That doesn’t seems relevant to software engineer apprentices

Reply:
ok I will remove it.

Reply:
startup owners will be good I guess? cause networking with various startup companies is important